test/hack/checkLabels/main.go

```diff
@@ -13,6 +13,13 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+/*
+* GLOSSARY
+* sourceMatchers and targetMatchers are the labels defined in the alertmanager config file
+* sourceLabels are the labels defined in the alerting rules from which originate the inhibitions
+* 	--> For example, 'scrape_timeout' is the source label for the 'cancel_if_scrape_timeout' inhibition label
+* cancelLabels is another name for the inhibition labels (for example : 'cancel_if_scrape_timeout')
+**/
```

Comment:
Maybe all a link to our doc (which is not yet super clear either, I have to admit):
https://github.com/giantswarm/prometheus-rules/#inhibitions

Also, if `sourceMatchers and targetMatchers are the labels defined in the alertmanager config file`, maybe naming them `am_sourceMatchers` and `am_targetMatchers` would clarify where they come from?

Reply:
True, I'll add this and change the labels' name

Reply:
Done